rotkehlchen/api/v1/resources.py

```diff
@@ -467,6 +467,7 @@ def get(self) -> Response:
 class EthereumTransactionsResource(BaseMethodView):
     get_schema = EthereumTransactionQuerySchema()
     post_schema = EthereumTransactionDecodingSchema()
+    put_schema = AsyncQueryArgumentSchema()
```

Comment:
I am not sure if this is the best approach. Semantically this sounds wrong.

You are doing a `PUT /blockchains/ETH/transactions`. This is hard to imagine as being a request to decode any undecoded ethereum transactions. I would expect this to add a transaction.

I would prefer a new endpoint for this `/blockchains/ETH/transactions/decode`